### ISSUE TITLE
Only having the universal audio stream

### ISSUE THREAD

**Issue by dglwilkins** [mdhiggins/sickbeard_mp4_automator#1414]
Some of the files I am trying to convert have 5.1 surround sound. I don't need this and only want a single 2 channel stereo stream. Converting it I seem to end up with both the 5.1 & stereo streams. I've played around with the autoProcess.ini file but can't seem to get what I want.

Does anyone know what setting I need to get just a single 2 channel Stereo audio stream?

**Comment by mdhiggins** [comment]
Don't use universal audio, that's designed to create an additional audio stream. Just leave that disabled and set your max-channels to 2 in the regular audio section

**Comment by tj1816** [comment]
I'm having this issue as well.  How do I disable universal audio?

**Comment by dglwilkins** [comment]
> I'm having this issue as well. How do I disable universal audio?

I'm terrible and never replied with what I found to fix it. I **think** I just removed the codec in the autoProcess.ini file so it looks like this:

[Universal Audio]
codec = 
channel-bitrate = 128
first-stream-only = True
move-after = False
filter = 
force-filter = False

I probably set the max channels to 2 like mdhiggins said as well. Can't 100% remember.
